packages/grid/x-grid-modules/src/hooks/features/useComponents.tsx

```diff
@@ -22,7 +22,7 @@ export const useComponents = (
   apiRef: ApiRef,
   gridRootRef: RootContainerRef,
 ) => {
-  const componentParams: ComponentParams = React.useMemo(
+  const componentProps: ComponentProps = React.useMemo(
```

Comment:
These used to be render functions, but are components now, hence `params` -> `props` to avoid confusion.